### DISCUSSION TITLE
poe check fails on main branch

### DISCUSSION THREAD

**Discussion by hsm207** [microsoft/autogen#4654] [Q&A] [answered]
I'm following the [development](https://github.com/microsoft/autogen/blob/e72549c298a5d4b12faf123ddfdf1b6c7e0d755b/python/README.md#development) instructions on the README and when I run `poe check` it fails with:

> Running task mypy in packages/autogen-ext
> Poe => mypy --config-file ../../pyproject.toml --exclude src/autogen_ext/runtimes/grpc/protos --exclude tests/protos src tests
> Success: no issues found in 53 source files
> Running task mypy in packages/autogen-magentic-one
> Poe => mypy --config-file /workspaces/alpha-oracle/third_party/autogen/python/packages/autogen-magentic-one/../../pyproject.toml src tests
> Success: no issues found in 29 source files
> Running task mypy in packages/autogen-agentchat
> Poe => mypy --config-file /workspaces/alpha-oracle/third_party/autogen/python/packages/autogen-agentchat/../../pyproject.toml src tests
> Success: no issues found in 46 source files
> Running task mypy in packages/autogen-core
> Poe => mypy --config-file ../../pyproject.toml --exclude src/autogen_core/application/protos --exclude tests/protos src tests
> Success: no issues found in 86 source files
> Poe => nbqa mypy docs/src --config-file ../../pyproject.toml
> Success: no issues found in 38 source files
> Running task mypy in packages/autogen-test-utils
> Poe => mypy --config-file /workspaces/alpha-oracle/third_party/autogen/python/packages/autogen-test-utils/../../pyproject.toml src tests
> mypy: can't read file 'tests': No such file or directory
> Error: Sequence aborted after failed subtask 'mypy'

Is the README missing an additional setup step?

**Comment by jackgerrits** [comment, marked as answer]
There's an issue for this: https://github.com/microsoft/autogen/issues/4597

It's easy to fix though, I am pretty sure we just need to override the mypy task in the failing package since it doesn't have a test dir.

**Comment by jackgerrits** [comment]
Issue has now been resolved
